webpack.config.js

```diff
@@ -43,16 +58,11 @@ var path = require('path'),
                 $: 'jquery'
             }),
             new Webpack.IgnorePlugin(/^(config.js)$/),
+            new Webpack.LoaderOptionsPlugin({
+                debug: true
+            }),
             new ExtractTextPlugin('ui-toolkit.css')
         ],
-        sassLoader: {
-            includePaths: [
-                path.resolve(__dirname, './node_modules'),
-                path.resolve(__dirname, './node_modules/edx-pattern-library/node_modules')
-            ],
-            data: '$pattern-library-path: \'' + siteRoot + './public/edx-pattern-library\' !default;'
-        },
-        debug: true,
```

Comment:
Moved this directly to the loader config above per https://webpack.js.org/guides/migrating/#loader-configuration-is-through-options